tests/storage-metadata

```diff
@@ -72,6 +72,8 @@ else
     fi
 fi
 
+apt install pigz --no-install-recommends
```

Comment:
yep sounds good

Reply:
Sorry the action suggestion was for `lxd`, not `lxd-ci`. For `lxd-ci`: `install_deps pigz`